.github/ISSUE_TEMPLATE/bug_report.yaml

```diff
@@ -0,0 +1,74 @@
+name: Bug report
+description: Create a report to help us improve
+labels: ["bug", "triage:needs-triage"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this bug report! Please make sure to fill out the entire form below, providing as much context as you can in order to help us triage and track down your bug as quickly as possible.
+
+        Before filing a bug, please be sure you have searched through [existing bugs](https://github.com/open-telemetry/semantic-conventions/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc+label%3Abug) to see if an existing issue covers your bug.
+  - type: dropdown
+    id: area
+    attributes:
+      label: Area(s)
+      description: Which area(s) does your bug report concern? If none fits, please select `area:other`
+      multiple: true
+      options:
+        - area:other
+        # NOTE: The list below is autogenerated using `make generate-gh-issue-templates`
+        # DO NOT manually edit it.
+        # Start semconv area list
+        - area:browser
+        - area:client
+        - area:cloud
+        - area:code
+        - area:container
+        - area:db
+        - area:destination
+        - area:device
+        - area:disk
+        - area:error
+        - area:exception
+        - area:host
+        - area:http
+        - area:k8s
+        - area:messaging
+        - area:network
+        - area:oci
+        - area:os
+        - area:process
+        - area:rpc
+        - area:server
+        - area:source
+        - area:thread
+        - area:tls
+        - area:url
+        - area:user-agent
+        # End semconv area list
+  - type: textarea
+    attributes:
+      label: What happened?
+      description: Please provide as much detail as you reasonably can.
+      value: |
+        ## Description
+
+        ## Steps to Reproduce
```

Comment:
nit: do we need this item? we don't really have anything runnable or reproducible

Reply:
I was thinking on the case where someone uses an attribute, maybe update the semconv version, and then something goes wrong. That could be a "reproducible" situation. But fine also to remove it.

Reply:
we can always remove it later depending on how useful it would be, definitely not blocking.